less/index.less

```diff
@@ -67,110 +74,244 @@ h1, h2, h3, h4, h5 {
 .container {
     width: 880px;
     margin: auto;
+    max-width: 880px;
 }
 
-/* Header Styles */
+// Header Styles
 
 header {
     width: 100%;
     display: flex;
-}
-
-header nav ul {
-    margin-top: 40px; 
-    width: 610px;
-    justify-content: space-between;
-    display: flex;
-}
+    nav{
+        .dropdown{
+            display:none;
+        }
+        ul {
+            margin-top: 40px; 
+            width: 610px;
+            justify-content: space-between;
+            display: flex;
+            
+            a {
+                text-decoration: none;
+                color: #b0b0b0;
+            }
 
-header nav ul li a {
-    text-decoration: none;
-    color: #b0b0b0;
+            a:hover {
+                text-decoration: underline;
+                color: black;
+            }
+        }
+        .logo {
```

Comment:
`.logo` is a sibling of `nav` and should be outside its block!

Nice use of `filter: invert()` though! 👍